util.go

```diff
@@ -28,10 +28,9 @@ func peerToPeerInfo(p *pb.CircuitRelay_Peer) (pstore.PeerInfo, error) {
 	addrs := make([]ma.Multiaddr, len(p.Addrs))
 	for i := 0; i < len(addrs); i++ {
 		a, err := ma.NewMultiaddrBytes(p.Addrs[i])
-		if err != nil {
-			return pstore.PeerInfo{}, err
+		if err == nil {
+			addrs[i] = a
```

Comment:
This will cause the `addrs` to have gaps between addresses. Are you 100% sure it is ok?

Reply:
Good catch. I believe so but we still shouldn't do this.